### PR TITLE
Fixes orientation overflow on event imports page.

### LINKED ISSUE #71: Avoid overflow of the pagination item on event imports page.
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Current : 
overflow

Expected : 
Overflow of pagination component to be avoided on small screens.

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #71 

Before : 
overflow

After : 
after

### CODE REVIEW

**Comment by sumedh123** [comment]
@niranjan94 Removing only that class aligns it to the left.

**Comment by niranjan94** [comment]
@sumedh123 on smaller screens left is acceptable. In fact, in most smaller screens once there are multiple pages, it would really matter since the pagination would take up the full space.

File: app/templates/components/events/imports-history-section.hbs [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <tfoot id = "pagination-bar">
       <tr>
         <th colspan="3">
-          <div class="ui right floated pagination menu">
+          <div class="ui pagination menu {{unless device.isMobile 'ui pagination menu right floated'}}">
```
Comment (niranjan94) [inline app/templates/components/events/imports-history-section.hbs]: Why are `pagination menu` classes duplicated ?

File: app/templates/components/events/imports-history-section.hbs [hunk 2]
```diff
@@ -32,10 +32,10 @@
         <td>{{t '8 days ago'}}</td>
       </tr>
     </tbody>
-    <tfoot>
+    <tfoot id = "pagination-bar">
```
Comment (niranjan94) [inline app/templates/components/events/imports-history-section.hbs]: Remove this Id.

**Pull request merged**